modules/test_user_dashboard/spec/services/create_test_user_account_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TestUserDashboard::CreateTestUserAccount do
+  subject { described_class.new }
+
+  file_path = Rails.root.join('modules', 'test_user_dashboard', 'spec', 'support', 'spec_users.csv')
```

Comment:
@bramleyjl We should convert line 8 and 9 to be:

```ruby
let(:file_path) {  File.join(Rails.root, 'modules/test_user_dashboard/spec/support/spec_users.csv') }
let(:users).     { CSV.read(file_path, headers: true) } 

subject { described_class.new }
```

I typically don't test initializers but that's mostly because I don't believe anything should really happen in them but assignment.